docs_src/design/ucr/0001-System-Events-for-Devices.md

```diff
@@ -0,0 +1,49 @@
+## System Events for Devices
+### Submitters
+- Lenny Goodell (Intel)
+
+### Change Log
+- [pending](https://github.com/edgexfoundry/edgex-docs/pull/779) (2022-06-22)
+
+### Market Segments
+- All or Any segments that can take advantage of auto on-boarding of new devices and auto start some AI/ML processing on the data from the new devices.
+
+### Motivation
+When a new camera device is added to the system only Core Metadata and the Device Service managing the new camera device are aware. There are use cases when other parts of the  system need to know when a new device has been added to the system. This UCR will focus on the camera management uses case which illustrates the need for these new System Events.
```

Comment:
Lots of other places that have been discussed as to where these types of events could be used.
- when a device/sensor is removed (as well as added/provisioned)
- when a device/sensor first starts to report data or when it stops sending data (the fact that the device service doesn't have the means to detect these yet doesn't lessen the fact that a central alerting capability is needed)
- When a new service is registered/when a service is unregistered
- When a scheduled operation kicked off (like clean up of the DB)
- Anytime an error is logged in a service

Reply:
Your first bullet is addressed by this UCR which specifically is about Device System Events. Your second bullet needs its own UCR for adding the `starts to report data` capability before you can have it included in a UCR for System Events. The remaining bullets are other System Events use cases that could be documented in UCRs, but this UCR is specifically focused on the Device System Events which we have a specific need for. IMO, adding these others muddies the water. I want to keep this to a tangible UC need that we can then build on later.